.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+job-defaults: &job-defaults
+  docker:
+    - image: circleci/node:12.1.0
+
+version: 2
+jobs:
+  prepare:
+    <<: *job-defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install packages
+          command: yarn --pure-lockfile --no-progress
+      - save_cache:
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
+      - run:
+          name: Prepare .env
+          command: |
+            echo NODE_ENV=production >> .env
+            echo BRANDGILITY_URL=n/a >> .env
```

Comment:
see this with @davesykeselateral

Reply:
@hhastekin-elateral we can merge this PR and decide `BRANDGILITY_URL` later

Reply:
@sunstorymvp yes, lets do that